code/WorkInProgress/MechanicMadness.dm

```diff
@@ -2413,16 +2413,22 @@
 		frequency = new_frequency
 		radio_connection = radio_controller.add_object(src, "[frequency]")
 		tooltip_rebuild = 1
-	proc/hear_radio(mob/M as mob, msg, lang_id)
+	proc/hear_radio(atom/movable/AM as mob|obj, msg, lang_id)
```

Comment:
Unless this is used as a verb, the 'as X' is useless (and might decrease performance?) iirc

Reply:
removed